### PR TITLE
Taskpage2

### LINKED ISSUE #20: Frontend: Implement Task List Page
Create an Task List page consistent with the UI in Figma. The list should be sortable and filterable according to our User Stories and Functional Requirements.

### PR DESCRIPTION
**Pull request by geoffblay**
General description of changes:
* Task page for properties, not individualized/dynamic yet
* New font family!

Breaking changes: 

Did you run `npm run before-pr`?
- [x] yes
- [ ] no

Does this PR fix/implement an Issue?
- [x] yes, fixes #20 
- [ ] no

Other notes:

### CODE REVIEW

**Review by bri-personal — CHANGES_REQUESTED** [review]
we already have an assets folder so let's put the images in there instead of making a new folder. otherwise it looks great!

File: public/index.html [hunk 1]
```diff
@@ -24,7 +24,13 @@
...
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Casamico</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@600;400&display=swap" rel="stylesheet">
+    <style>
+      @import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@600;400&display=swap');
+    </style>
+    <title>React App</title>
```
Comment (MrEminent42) [inline public/index.html:33]: Merge conflicts are tough. You can see in the Pull Request here that I changed the title from `React App` to `Casamico`. Could you revert this change (just manually change the title again)?

p.s. this kind of thing is super easy to miss during merge conflicts so don't worry about it

File: src/pages/Page2.tsx [hunk 2]
```diff
@@ -1,9 +1,130 @@
 import React from 'react'
+import styled from 'styled-components';
+import backbuttonsvg from '../images/arrow-left-circle.svg';
+import house from "../images/house.jpg";
+import addbuttonsvg from "../images/plus-button.svg";
+import FilterButton from '../components/FilterButton';
+import Task from '../components/Task';
 
 const Page2 = () => {
```
Comment (MrEminent42) [inline src/pages/Page2.tsx:9]: Rename component (and file) to `Tasks` or `TaskPage`?

**Review by MrEminent42 — CHANGES_REQUESTED** [review]

**Comment by MrEminent42** [comment]
Also please edit the original message to say `yes, fixes #20` to `Does this PR fix/implement an Issue`. That way, the issue will automatically close when this PR is merged!

File: package.json [hunk 3]
```diff
@@ -44,5 +43,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "cross-env": "^7.0.3"
```
[the remark this answers is not in the document]
Comment (geoffblay) [reply package.json:48]: yeah I didn't have cross-env installed locally so I had to install it myself, will this break anything?

[the remark this answers is not in the document]
Comment (MrEminent42) [reply package.json:48]: doesn't look like it...

File: src/App.tsx [hunk 4]
```diff
@@ -24,7 +24,7 @@ function App() {
           {/* Again, pass the state and the function to update it. */}
           <Route path="/" element={<Home setSelectedProperty={setSelectedProperty} />} />
           <Route path="/1" element={<Page1 />} />
-          <Route path="/2" element={<Page2 />} />
+          <Route path="/2" element={<Tasks />} />
```
Comment (MrEminent42) [inline src/App.tsx:27]:
```suggestion
          <Route path="/tasks" element={<Tasks />} />
```

**Review by MrEminent42 — APPROVED** [review]
Killing it!

**Comment by zachstoney30** [comment]
🖕

**Review by zachstoney30 — APPROVED** [review]
meh
